### ISSUE TITLE
FEATURE REQUEST: Add cmdlet to send toasts system-wide

### ISSUE THREAD

**Issue by bcurran3** [Windos/BurntToast#117]
REF: https://github.com/Windos/BurntToast/issues/33 and  https://github.com/Windos/BurntToast/issues/35

There are many instances where sending a toast message to any and all logged in users by a remote sysadmin would be useful.

I propose a new cmdlet, alias, -computer switch that would implement this.

Working proof of concept:
https://github.com/bcurran3/BurntToast/blob/bcurran3-patch-2-global-toast/BurntToast/Public/New-GlobalBurntToastNotification.ps1

![image](https://user-images.githubusercontent.com/14026600/93723721-e4f0ce00-fb55-11ea-8ef2-ce0ab5089e00.png)

This POC cmdlet works globally on the locally run system using $env:COMPUTERNAME. Changing it to a remote NETBIOS computer name will pop up the toast notification on that computer if WinRM is running on it. This POC cmdlet is not designed to test for WinRM status on remote computers. 

p.s. My skills are not refined enough to implement via a PR.

**Comment by bcurran3** [comment]
OK, I couldn't leave well enough alone and modified New-GlobalBurntToastNotification.ps1 to take a -computername switch to further along the POC. No doubt my ADD will kick in more and commit me to change the WinRM checking to work over the network....

**Comment by steviecoaster** [comment]
@Windos if you want to grab some time I can implement this as part of `New-BurntToastNotification` directly so there's no need for a secondary cmdlet to do this over the network. Would be good to discuss some implementation details before I get to work here though.

**Comment by temeyers** [comment]
Was this ever implemented ?

**Comment by SirPrieto** [comment]
Is this working without install the module on each machine?

**Comment by lexcorp** [comment]
Can this be used with the latest version?

**Comment by bcurran3** [comment]
@lexcorp Yes, works fine with v0.8.5. I have it implemented via some Chocolatey add-ons I created:
https://community.chocolatey.org/packages/chocolatey-toast-notifications.extension
https://community.chocolatey.org/packages/choco-toast-notifications.hook

**Comment by LukeShootsThings** [comment]
I just wanted to drop a comment saying this would be a valuable feature if it could be implemented.  I'd love to be able to utilize something like this without having to load the powershell module on each endpoint.
